server.go

```diff
@@ -316,6 +323,8 @@ func (s *Server) newEventCh(ctx context.Context) chan events.Event {
 	eventCh := make(chan events.Event)
 
 	logging.GoAnnotated(ctx, func(ctx context.Context) {
+		defer s.panicHandler.HandlePanic()
```

Comment:
Is this supposed to be added to every go routine?

Reply:
I am not sure how much we need to cover